src/amount.h

```diff
@@ -17,7 +17,7 @@ typedef int64_t CAmount;
 static const CAmount COIN = 100000000;
 static const CAmount CENT = 1000000;
 
-extern const std::string CURRENCY_UNIT;
```

Comment:
AFAIK this will create a separate std::string for every .cpp file this is included in.  This is inefficient.